app/assets/stylesheets/govuk_publishing_components/components/helpers/_brand-colours.scss

```diff
@@ -96,3 +96,27 @@
     border-color: #045f71;
   }
 }
+
+// This change should be removed after relevant govuk-frontend release
+
+.brand--department-for-business-and-trade {
+  .brand__color {
+    color: #005ea5;
+
+    &:link,
+    &:visited,
+    &:active {
+      color: #005ea5;
+    }
+
+    &:hover,
+    &:focus {
+      color: #005ea5;
```

Comment:
I think this line needs to change to match the example on line 70. We only want the colour to be applied to the link when it is not focussed, because the focus state has a different background colour (yellow) and therefore the contrast is different - for example, if the brand colour was yellow, the link when focussed would be yellow on yellow, and therefore unreadable. This is what it looks like now:

![Screenshot 2023-04-24 at 11 19 26](https://user-images.githubusercontent.com/861310/233969574-ec26dcf9-1194-453d-bca1-99c2275b34a3.png)

and this is what it should look like:

![Screenshot 2023-04-24 at 11 22 28](https://user-images.githubusercontent.com/861310/233969661-17ca24cb-5638-4217-abeb-c981047287cf.png)